docs/install.md

```diff
@@ -4,129 +4,164 @@ Kubernetes 1.7 or higher clusters run the
 [API Aggregator](https://kubernetes.io/docs/concepts/api-extension/apiserver-aggregation/),
 which is a specialized proxy server that sits in front of the core API Server.
 
-The aggregator allows user-defined, Kubernetes compatible API servers to come
-and go inside the cluster, and register themselves on demand to augment the
-externally facing API that Kubernetes offers.
+Service Catalog provides an API server that sits behind the API aggregator, 
+so you'll be using `kubectl` as normal to interact with Service Catalog.
 
-Instead of requiring the end-user to access multiple API servers, the API
-aggregation system allows many API servers to run inside the cluster, and
-combines all of their APIs into one externally facing API.
+To learn more about API aggregation, please see the 
+[Kubernetes documentation](https://kubernetes.io/docs/concepts/api-extension/apiserver-aggregation/).
 
-This system is very useful from an end-user's perspective, as it allows the
-client to use a single API endpoint with familiar, consistent tooling,
-authentication and authorization.
+The rest of this document details how to:
 
-The Service Catalog utilizes API aggregation to present its API.
+- Set up Service Catalog on your cluster
+- Interact with the Service Catalog API
 
 # Step 1 - Prerequisites
 
-## Starting Kubernetes with DNS
+## Kubernetes Version
 
-You *must* have a Kubernetes cluster with cluster DNS enabled. We can't list
-instructions here for enabling cluster DNS for all Kubernetes distributions, but
-here are a few notes:
+Service Catalog requires a Kubernetes cluster v1.7.0 or later. You'll also need a 
```

Comment:
Why include the patch level in the version number string? `v1.7 or later` looks better to me and is the same thing.